zeus/device/gpu.py

```diff
@@ -9,6 +9,8 @@
 
 import pynvml  # necessary for testing to mock!
 
+# import amdsmi
+
```

Comment:
Just checking my understanding: Since `amdsmi` has no PyPI package, pretty much the only way to install it is to `cd` into an existing ROCm installation and do `pip install .`. So `import amdsmi` won't work on a platform without AMD GPUs. Is this correct?

Reply:
Yes, that is correct